test/serializable_resource_test.rb

```diff
@@ -71,6 +71,13 @@ def test_serializable_resource_with_collection_containing_errors
         }
         assert_equal serializable_resource.as_json(options), expected_response_document
       end
+
+      def test_serializable_resource_when_no_serializer_found
+        resource = ModelWithErrors.new
+        serializable_resource = ActiveModel::SerializableResource.new(resource)
+        # TODO define error to raise here
+        assert_raises { serializable_resource.as_json }
```

Comment:
oh, the `NoSerializerError` is just an unfortunate part of our api right now. It's real.

I figure OP might want to rebase this with changes like that and force push.  I just wrote that up in the browser, might not be precise